stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -184,6 +185,8 @@ pub struct StacksHeaderInfo {
     /// The burnchain tip that is passed to Clarity while processing this block.
     /// This should always be `Some()` for Nakamoto blocks and `None` for 2.x blocks
     pub burn_view: Option<ConsensusHash>,
+    /// Total tenure size (reset at every tenure extend) in bytes
```

Comment:
You might also want to add a comment here that this value is not consensus-critical and may differ between nodes.